fuzz/ocsp.cc

```diff
@@ -0,0 +1,26 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <openssl/err.h>
+#include <openssl/mem.h>
+#include <openssl/ocsp.h>
+
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len) {
+  OCSP_RESPONSE *ocsp_response = d2i_OCSP_RESPONSE(NULL, &buf, len);
+  if (ocsp_response != NULL) {
+    // Extract the basic response
+    OCSP_BASICRESP *basic_response = OCSP_response_get1_basic(ocsp_response);
+    if (basic_response != NULL) {
+      OCSP_BASICRESP_free(basic_response);
+    }
+
+    // Reserialize the structure.
+    uint8_t *der = NULL;
+    i2d_OCSP_RESPONSE(ocsp_response, &der);
+    OPENSSL_free(der);
+  }
+  OCSP_RESPONSE_free(ocsp_response);
+  ERR_clear_error();
+  return 0;
+}
```

Comment:
Some questions:

1. Is the input (`const uint8_t *buf, size_t len`) from `fuzz/ocsp_corpus` dir?

2. Is this file used to tell fuzz how to test OCSP related functions (e.g. `d2i_OCSP_RESPONSE`, `OCSP_response_get1_basic`)?

Reply:
1. Yes, this input reads from directory we give to the fuzzing command. In this case, it's `fuzz/ocsp_corpus`.
2. Yes, this file is how we tell Libfuzzer to parse our inputs. I'll clarify that in the runbook guide.